<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Nestor Rodriguez</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>209</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cga</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="paramBox">
      <property name="title">
       <string>Parameters</string>
      </property>
      <zorder>verticalLayoutWidget</zorder>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="resultBox">
      <property name="title">
       <string>Result</string>
      </property>
      <widget class="QTextEdit" name="resultTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>paramBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
